### ISSUE TITLE
[Bug] editing task in progress throws an exception

### ISSUE THREAD

**Issue by Link-** [TailorDev/Watson#417]
For some reason the editing of tasks in progress is broken [master](https://github.com/TailorDev/Watson/tree/33013239755ff8c4438f64fbd299ddec6f07a56e).

```sh
Watson on 🌱 master [🤷] via 🐍 v3.8.8 (.venv)
13:17:24❯ watson start Project +"New work item"
Starting project Project [New work item] at 13:17

Watson on 🌱 master [🤷] via 🐍 v3.8.8 (.venv)
13:17:34❯ watson edit
Error: Error converting date: Cannot parse argument of type None.

Watson on 🌱 master [🤷] via 🐍 v3.8.8 (.venv)
13:17:36❯ watson stop
Stopping project Project [New work item], started just now and stopped just now. (id: ba10b67)

Watson on 🌱 master [🤷] via 🐍 v3.8.8 (.venv)
13:17:41❯ watson edit
Edited frame for project Project [New work item], from 13:17:34 to 13:17:40 (06s)
```

I believe the issue is in the following code:

https://github.com/TailorDev/Watson/blob/33013239755ff8c4438f64fbd299ddec6f07a56e/watson/cli.py#L1283

https://github.com/TailorDev/Watson/blob/33013239755ff8c4438f64fbd299ddec6f07a56e/watson/frames.py#L11-L25

`stop` needs to be handled as an optional parameter in the Frame constructor. This doesn't look like new code and I'm not sure how this is working in previous releases 🤷‍♂️ maybe the upgrade to `arrow>=1.0.0` has something to do with it?

**Comment by jmaupetit** [comment]
I think you are right: this is related to the recent arrow upgrade... and not only `master` but the latest `2.0.0` release is broken!

Since it broke #388 do you have time to take a look at it? :pray:

**Comment by Link-** [comment]
@jmaupetit I already did, if we're going to maintain this version of arrow the fix is not simple. We have 2 options:

1. Update the `Frame` class to support instantiation with optional `stop` parameter
2. Define `stop` as the `arrow.utcnow()` every time we make an edit to an on-going frame like this:
https://github.com/Link-/Watson/blob/1352c2dfd31c4d9e8cca88f03a6f80ed57e97d6b/watson/cli.py#L1284-L1358

I think option 1 is the cleanest but before that I'm not really sure `why` the upgrade broke it.

**Comment by jmaupetit** [comment]
I vote for the first option. Thank you for this investigation.

**Comment by komputerwiz** [comment]
I ran into this earlier today after updating. After finding and reading the conversation here, I decided to take a stab at a potential solution.

Thanks for the great work!

**Comment by jmaupetit** [comment]
Fixed in #418
